app/boards/arm/kbdfans_tofu65_v2/kbdfans_tofu65_v2.dts

```diff
@@ -0,0 +1,131 @@
+/*
+ * Copyright (c) 2023 The ZMK Contributors
+ * SPDX-License-Identifier: MIT
+ */
+
+/dts-v1/;
+
+#include <rpi_pico/rp2040.dtsi>
+#include <freq.h>
```

Comment:
```suggestion
```

Shouldn't be needed any more.

Reply:
At one point I thought I needed this to complete the build. I guess what ever it was is now gone.